core/gui/src/app/common/service/user/auth.service.ts

```diff
@@ -103,11 +103,8 @@ export class AuthService {
 
     const role = this.jwtHelperService.decodeToken(token).role;
     const email = this.jwtHelperService.decodeToken(token).email;
-
     if (this.inviteOnly && role == Role.INACTIVE) {
-      this.notificationService.loading("The account request of <b>" + email + "</b> is received and pending.", {
-        nzDuration: 10,
-      });
+      alert("The account request of " + email + " is received and pending.");
```

Comment:
why do we use alerts instead of using notificationService?

Reply:
notificationService doesn't work due to the route change.